### DISCUSSION TITLE
Making Translation Easier

### DISCUSSION THREAD

**Discussion by Poseidon281** [pelican-dev/panel#52] [Ideas]
Is it a good idea to make a Crowdin Project? So people can easily translate the project?

**Comment by Poseidon281** [comment]
I made a crowdin page for Translations: https://crowdin.com/project/pelican-panel

**Reply by martinbndr** [reply]
Wouldn´t it be better to have this be created by the maintainers itself so it can be integrated in their github?
Then they can also decide which translations should be merged into it etc

**Reply by Boy132** [reply]
There is now an official crowdin project: https://crowdin.com/project/pelican-dev

**Comment by notAreYouScared** [comment]
The Crowdin project can be found here: https://crowdin.com/project/pelican-dev
